klient/decoder/decoder.go

```diff
@@ -217,6 +218,18 @@ func DecodeFile(fsys fs.FS, manifestPath string, obj k8s.Object, options ...Deco
 	return Decode(f, obj, options...)
 }
 
+// DecodeURL is a stream of documents from the URL of any Kind using either the innate typing of the scheme.
```

Comment:
I would reword the start of the comment `// DecodeURL decodes a document from the URL...`

Reply:
Thanks, fixed.